src/System.IO.FileSystem/tests/FileStream/ctor_str_fm_fa_fs_buffer_fo.cs

```diff
@@ -66,6 +66,16 @@ public void ValidFileOptions(FileOptions option)
             }
         }
 
+        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
```

Comment:
ValidFileOptions test is excluded on Nano Server. I call it to test Encrypted option so I excluded ValidFileOptions_Encrypted test on Nano Server also.